docs/developer-docs/functionality/sns/sns.md

```diff
@@ -1,2 +1,230 @@
-# Handing over to SNS
+# SNS
 
+A decentralized autonomous organization, or DAO for short, is a system that allows
+many parties to jointly control an entity.
+Similarly to how the Network Nervous System (NNS) is the open tokenized DAO that controls
+the Internet Computer blockchain (IC), service nervous systems 
+(SNSs) are algorithmic DAOs that allow developers to create decentralized, 
+token-based governance systems for their dapps.
+This means, each dapp that would like to be under decentralized control will have a 
+separate SNS.
```

Comment:
I agree somewhat but I also think it is important to clarify this (as otherwise people might thing "1 NNS, 1 SNS", so for now I would leave it in as there don't seem to be counter-proposals.